<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowView</class>
 <widget class="QWidget" name="MainWindowView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>352</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KLineEdit" name="titleEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KLineEdit" name="descriptionEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="authorLabel">
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="KLineEdit" name="authorEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="versionLabel">
          <property name="text">
           <string>Version:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="KLineEdit" name="versionEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="dateLabel">
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="dateEdit">
          <property name="displayFormat">
           <string>d/M/yyyy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="maintypeLabel">
          <property name="text">
           <string>Main Type:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KComboBox" name="maintypeBox">
          <item>
           <property name="text">
            <string>image</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>relation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sound</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>word</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="backLabel">
          <property name="text">
           <string>Back Image:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KUrlRequester" name="backKurl"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="pixLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-add">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-remove">
             <normaloff/>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KComboBox" name="moreButton">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="minimumContentsLength">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>New Image</string>
            </property>
            <property name="icon">
             <iconset theme="view-preview">
              <normaloff/>
             </iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Logic</string>
            </property>
            <property name="icon">
             <iconset theme="preferences-plugin">
              <normaloff/>
             </iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Sound</string>
            </property>
            <property name="icon">
             <iconset theme="preferences-desktop-text-to-speech">
              <normaloff/>
             </iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Word</string>
            </property>
            <property name="icon">
             <iconset theme="preferences-desktop-font">
              <normaloff/>
             </iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string>Image:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KUrlRequester" name="fileKurl"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="wordLabel">
         <property name="text">
          <string>Word:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="wordEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="langLabel">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>en</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>de</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>it</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>es</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pt</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pl</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cz</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="itemLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
